pyproject.toml

```diff
@@ -1,3 +1,39 @@
+[tool.poetry]
+name = "sceptre-resolver-template"
+version = "0.0.1"
+packages = [{ include = "resolver" }]
+readme = "README.md"
+homepage = "https://github.com/Sceptre/sceptre-resolver-template"
+repository = "https://github.com/Sceptre/sceptre-resolver-template"
+authors = ["Sceptre <sceptreorg@gmail.com>"]
+description = "A Github template for Sceptre resolvers"
+keywords = ["sceptre", "sceptre-resolver"]
+license = "Apache-2.0"
+classifiers = [
+  "Intended Audience :: Developers",
+  "Natural Language :: English",
+  "Environment :: Console",
+  "Programming Language :: Python :: 3.8",
+  "Programming Language :: Python :: 3.9",
+  "Programming Language :: Python :: 3.10"
+]
+
+[tool.poetry.plugins."sceptre.resolvers"]
+"custom" = "resolver.custom:Custom"
+
+[tool.poetry.dependencies]
+python = "^3.8.1"
+sceptre = { version = "^3.2.0", optional = true }
```

Comment:
This means ">3.2,<4.0". I don't believe that's what you want here:

https://python-poetry.org/docs/dependency-specification/

Also, why is sceptre an optional dependency?

Reply:
i have changed it to a dev dependency